csv-pricing.md

```diff
@@ -0,0 +1,133 @@
+
+Provide a file path for your CSV pricing data in [values.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values-custom-pricing.yaml). This path can reference a local PV or an S3 bucket.
+
+```yaml
+pricingCsv:
+    enabled: true
+    location:
+    provider: "AWS|GCP"
+    region: "us-east-1"
+    URI: s3://YOUR_BUCKET/path/custom-pricing.csv
+    csvAccessCredentials: pricing-schema-access-secret
+```
+
+Alternatively, mount a configmap with the CSV:
+
+``` sh
+kubectl create configmap csv-pricing --from-file custom-pricing.csv
+```
+
+Helm values:
+
+``` yaml
+pricingCsv:
+  enabled: true
+  location:
+    URI: /var/kubecost-csv/custom-pricing.csv
+    location: local
+extraVolumes:
+- name: kubecost-csv
+  configMap:
+    name: csv-pricing
+extraVolumeMounts:
+- name: kubecost-csv
+  mountPath: /var/kubecost-csv
+```
+
+For S3 locations, provide file access. Required IAM permissions:
+
+```json
+    {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+           "Resource": [
+                "arn:aws:s3:::<your-bucket-name>/*",
+                "arn:aws:s3:::<your-bucket-name>"
+            ]
+        }
+      ]
+    }
+```
+
+There are two options for adding the credentials to the Kubecost pod:
+
+1. Service key: Create an S3 service key with the permissions above, then add its ID and access key as a K8s secret:
+   1. `kubectl create secret generic pricing-schema-access-secret -n kubecost --from-literal=AWS_ACCESS_KEY_ID=id --from-literal=AWS_SECRET_ACCESS_KEY=key`
+   2. The name of this secret should be the same as csvAccessCredentials in values.yaml above
+
+
+2. AWS IAM (IRSA) [service account annotation](https://docs.aws.amazon.com/eks/latest/userguide/adot-iam.html)
+
+## Pricing discounts
+
+Negotiated discounts are applied after cost metrics are written to Prometheus. Discounts will apply to all node pricing data, including pricing data read directly from the custom provider CSV pipeline. Additionally, all discounts can be updated at any time and changes are applied retroactively.
+
+## Pricing inference
+
+The following logic is used to match node prices accurately:
+
+* First, search for an exact match in CSV pipeline
+* If exact match not available, search for an existing CSV data point that matches region, instanceType, and AssetClass
+* If neither available, fallback to pricing estimates
+
+You can check a summary of the number of nodes that have matched with the CSV by visiting /model/pricingSourceCounts. The response is a JSON object of the form:
+
+``` jsonc
+data:
+{
+	TotalNodes: 10
+	PricingType: {
+		csvExact: 5 // exact matches by the providerID field
+		csvClass: 4 // matches where the region and instanceType match
+		"": 1 // matches that use our default pricing
+    }
+}
+```
+
+## Need Additional Help?
+
+Contact us
```

Comment:
@bstuder99 can you add / remove this based on what you think is required?

Reply:
I would just remove this part